main.py

```diff
@@ -73,9 +42,15 @@ def convert_website_to_stamp(_website, maximum_pages):
 
     url, maximum_pages = get_user_input()
 
-    _website = Website(url)
-
-    if not _website.is_valid:
-        LOGGER.error("Invalid URL!")
-    else:
-        convert_website_to_stamp(_website, maximum_pages)
+    _stampifier = Stampifier(url, maximum_pages)
+
+    try:
+        _stamp_file_path = _stampifier.stampify()
+    except InvalidUrlError as err:
+        LOGGER.debug(err.message, err.provided_url)
+    except NoneTypeMarkupError as err:
+        LOGGER.debug(err.message)
+    except WebsiteNotStampifiableError as err:
+        LOGGER.debug(err.message, "Error received by:", err.failure_source)
+    except WebsiteConnectionError as err:
+        LOGGER.debug(err.message, err.provided_url, err.trace)
```

Comment:
If every error has message, then we should simply for message. Let trace, url, source be part of message.
If you think message is meant for something else, you can log_statement in base error.

Reply:
Something like this:
```suggestion
    try:
        _stamp_file_path = _stampifier.stampify()
    except Error as err:
        LOGGER.debug(err.message)
```

Reply:
Updated! Let me know if the new changes resolve this.